content/updates/2018-jan.md

```diff
@@ -0,0 +1,21 @@
+---
+title: January 2018
+date: 2018-02-07
+hidden: true
+image: image  # pending pi-image
+update: update  # pending pi-image
+---
+
+- Support for [custom Arduino commands](/api/servo-assembly/arduino-commands/)
+- Implement [Cartesian coordinate system](/api/vision/coordinates/#cartesian-coordinates)
+- Fix issue with camera after code terminates
+- Remove dates from logs (They were wrong anyway)
+- Remove unnecessary noisy log messages
+- Remove unnecessary `size` attribute from a marker
+- Deprecate broken `polar` coordinate system in favour of [`spherical`](/api/vision/coordinates/#spherical-coordinates)
+- Flash the start LED once robot is ready (Also reduced chance of press not being registered)
```

Comment:
I think this "(Also ...)" is worth having as a separate point. While it was in the same bit of code, it's not really related from a user perspective. (Doing that also solves the grammar quandary it currently has).

Could we also clarify that it's the "start button press" that we're talking about?